test/setup/iam.tf

```diff
@@ -27,6 +27,7 @@ locals {
 
   int_required_folder_roles = [
     "roles/owner",
+    "roles/logging.admin",
```

Comment:
The new framework does not yet [recognize](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/39b4ef08bb23b352ea6ff7073f942ec0b5a50fc7/infra/build/developer-tools/build/scripts/export_tf_outputs.py#L108) SA output to [activate it](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/66fe6ccc4cf81c769179af638d08ce69cd66c57f/infra/build/developer-tools/build/scripts/task_helper_functions.sh#L439) automatically. I believe test maybe failing as its not activating the right SA. As a workaround you can do a `source_test_env && init_credentials` in prepare which should activate it.

Reply:
Thanks, do we have a tracking issue?

Reply:
opened https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/999 to track